DataFormats/simulation/src/Stack.cxx

```diff
@@ -365,6 +371,13 @@ void Stack::Print(Int_t iVerbose) const
   }
 }
 
+void Stack::Print(Option_t* option) const
+{
+  Int_t verbose = 0;
+  if ( option ) verbose = 1;
```

Comment:
Here the logic should be extended to actually check for specific option to set verbosity